nano/node/node.cpp

```diff
@@ -1745,24 +1749,29 @@ std::pair<uint64_t, decltype (nano::ledger::bootstrap_weights)> nano::node::get_
 	return { max_blocks, weights };
 }
 
-void nano::node::populate_backlog ()
+bool nano::node::populate_backlog ()
 {
+	std::cerr << "<---------------Populating...\n";
```

Comment:
Is a direct print to console needed here?

Reply:
I've done some testing on this PR and it looks really good
This is the baseline test (v23 RC3 branch)
![BaselineV23](https://user-images.githubusercontent.com/81099017/149372726-7d081911-11d0-463a-bf7f-468dd395dd76.png)
Cycling is obvious

This V23 RC3 bu with included cherry pick of this PR:
![V23+3648](https://user-images.githubusercontent.com/81099017/149372751-85b4a152-e843-4ff6-ae3e-1aad8742eb04.png)
No cycling and average CPS is approx three times higher